components/modals/settings/workers.tsx

```
@@ -1,11 +1,11 @@
-import { useCallback, useContext, useEffect, useMemo, useState } from "react"
-import { ProvisioningStatus } from "@/lib/schema-worker/SchemaGraph"
-import { CrateVerifyContext } from "@/components/providers/crate-verify-provider"
+import { Fragment, useCallback, useContext, useEffect, useMemo, useState } from "react"
+import { SchemaWorker } from "@/components/providers/schema-worker-provider"
 import { Error } from "@/components/error"
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)